metersink/api.py

```diff
@@ -20,29 +20,43 @@ def _init_logger():
 
 CONFIG = None
 
+def push_to_sinks(conf, data):
+    """puts received metering data to the configured billing sinks"""
+    # LOG.debug('%s', conf)
+    sinks = get_sinks(conf)
+    LOG.debug("pushing to sinks: %s", sinks)
+    for sink_type, sink_values in sinks.items():
+        if sink_type == "file":
+            for index, sink_name in enumerate(sinks["file"]["name"]):
+                # maybe we want to differ between events and polls here
+                # for now we put all incoming into the file
+                output_file(sink_name, data)
+        elif sink_type == "odoo":
+            odoo_handle(sinks['odoo'], conf, data)
+
 
 @app.route("/post_json", methods=["POST"])
 def process_json():
+    """Endpoint for json requests"""
     content_type = request.headers.get("Content-Type")
     if content_type == "application/json":
         json_data = request.json
-        LOG.debug("%s", pformat(json_data))
+        LOG.debug("### the request ###############################################################")
+        LOG.debug("json_body: %s", pformat(json_data))
         data = json.loads(request.data)
         for message in data:
-            LOG.debug("message: %s", pformat(message))
             push_to_sinks(CONFIG, message)
         return json_data, 202
-    else:
-        return "Content-Type not supported!", 204
+    return "Content-Type not supported!", 204
```

Comment:
It would be nice to repeat the content-type here (otherwise the variable should be inlined).